core/services/ocr2/delegate.go

```diff
@@ -771,6 +781,121 @@ func (d *Delegate) newServicesMercury(
 	return mercuryServices, err2
 }
 
+func (d *Delegate) newServicesLLO(
+	ctx context.Context,
+	lggr logger.SugaredLogger,
+	jb job.Job,
+	bootstrapPeers []commontypes.BootstrapperLocator,
+	kb ocr2key.KeyBundle,
+	ocrDB *db,
+	lc ocrtypes.LocalConfig,
+	ocrLogger commontypes.Logger,
+) ([]job.ServiceCtx, error) {
+	lggr = logger.Sugared(lggr.Named("LLO"))
+	spec := jb.OCR2OracleSpec
+	transmitterID := spec.TransmitterID.String
+	if len(transmitterID) != 64 {
+		return nil, errors.Errorf("ServicesForSpec: streams job type requires transmitter ID to be a 32-byte hex string, got: %q", transmitterID)
```

Comment:
why does the error message say 32 but the check is len == 64?

Reply:
Because 32 bytes makes a 64-character hex string